node/utils/errors.ts

```diff
@@ -0,0 +1,20 @@
+import { AuthenticationError, ForbiddenError, UserInputError } from '@vtex/api'
+import { AxiosError } from 'axios'
+
+export const statusToError = (e: any) => {
```

Comment:
I think this already exists on utils/index.ts

Reply:
removed thx